models/lpjguess/R/readStateBinary.LPJGUESS.R

```diff
@@ -208,13 +208,13 @@ readStateBinary <- function(out.path, npft){
   nofpatch <- readBin(zz, integer(), 1, size = 4)  
   
   for(pat in 1:nofpatch){
-    Patchpft_list[[pat]] <- getClass_Patchpft()
-    Vegetation_list[[pat]] <- getClass_Vegetation()
-    Individual_list[[pat]] <- getClass_Individual(Vegetation_list[[pat]]$indv)
-    Soil_list[[pat]] <- getClass_Soil()
-    Sompool_list[[pat]] <- getClass_Sompool()
-    SompoolCent_list[[pat]] <- getClass_SompoolCent() 
-    Fluxes_list[[pat]] <- getClass_Fluxes()
+    Patchpft_list[[pat]] <- getClass_Patchpft(zz,npft)
+    Vegetation_list[[pat]] <- getClass_Vegetation(zz)
+    Individual_list[[pat]] <- getClass_Individual(Vegetation_list[[pat]]$indv,zz)
```

Comment:
👍 to this solution to the scoping problem. Nit: Spaces after commas, please.

```suggestion
    Patchpft_list[[pat]] <- getClass_Patchpft(zz, npft)
    Vegetation_list[[pat]] <- getClass_Vegetation(zz)
    Individual_list[[pat]] <- getClass_Individual(Vegetation_list[[pat]]$indv, zz)
```